src/Schema/Index.php

```diff
@@ -186,11 +186,19 @@ public function spansColumns(array $columnNames)
     }
 
     /**
-     * Checks if the other index already fulfills all the indexing and constraint needs of the current one.
+     * Keeping misspelled function name for backwards compatibility
```

Comment:
Although we need to keep the method around in the current major release, it should be deprecated and removed in the next major.

Reply:
Yep, @greg0ire already mentioned that.

Reply:
@morozov I think this can be merged as is, then the deprecation can be added on 3.5.x

Reply:
@nexxai please don't mark as resolved conversations you do not open in general, and here in particular because it hides my comment.

Reply:
Oh no, I'm sorry!  I'll leave them open for the commenter to close.  Again, my apologies.

Reply:
No worries.